templates/test/result.html.ep

```diff
@@ -44,6 +44,14 @@
                     % } else
                     % {
                         State:<%= $job->state %>
+                        % if (scalar($job->gru_dependencies) > 0) {
+                            % my $grutext = '';
+                            % for my $grudep ($job->gru_dependencies) {
+                                % $grutext .= join(",", $grudep->gru_task->id, $grudep->gru_task->taskname);
+                                % $grutext .= " ";
+                            % }
+                            waiting for GRU tasks: <%= $grutext %>
```

Comment:
What about 'waiting for background tasks:'? I guess not many people now what GRU task is.